### PR TITLE
Add margin to WP dashboard notice copy (3911).

### PR DESCRIPTION
**Pull request by johnPhillips**
## Summary

Addresses issue #3911

## Relevant technical choices
Added margin to the notice copy to improve how it looks on smaller screens.
See discussion on the issue: https://github.com/google/site-kit-wp/issues/3911#issuecomment-913459933

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/components/idea-hub/_googlesitekit-idea-hub-wpdashboard-widget.scss [hunk 1]
```diff
@@ -42,6 +42,7 @@
 
 	.googlesitekit-idea-hub__wpdashboard--copy {
 		color: map-get($wp-colors, "blue-80");
+		margin-right: 14px;
```
Comment (aaemnnosttv) [inline assets/sass/components/idea-hub/_googlesitekit-idea-hub-wpdashboard-widget.scss:45]: This looks great but it ends up pushing the text to two lines on a 4-col dashboard (en-US of course).

image

@marrrmarrr how about we change the CTA to "View Ideas" instead?

image

Comment (aaemnnosttv) [reply assets/sass/components/idea-hub/_googlesitekit-idea-hub-wpdashboard-widget.scss:45]: @johnPhillips, @marrrmarrr confirmed that this change is okay to make. Would you please update this as well?

Comment (johnPhillips) [reply assets/sass/components/idea-hub/_googlesitekit-idea-hub-wpdashboard-widget.scss:45]: Of course, good idea 👍

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks @johnPhillips !

**Pull request merged**